<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FEMMaterialsDialog</class>
 <widget class="QDialog" name="FEMMaterialsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>325</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Edit FEM Materials</string>
  </property>
  <property name="windowOpacity">
   <double>0.950000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_36">
       <item>
        <widget class="QDoubleSpinBox" name="m_YoungsModulusMantissaSpin">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>-9.900000000000000</double>
         </property>
         <property name="maximum">
          <double>9.900000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_31">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
         </property>
         <property name="text">
          <string>e</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="m_YoungsModulusExponentSpin">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
         <property name="maximum">
          <double>99.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_PoissonsLabel">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Poisson's Ratio:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QDoubleSpinBox" name="m_PoissonsSpin">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.300000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_CrossSectionalLabel">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Cross Sectional Area:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QDoubleSpinBox" name="m_CrossSectionalSpin">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="12" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_ThicknessLabel">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Thickness:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="12" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QDoubleSpinBox" name="m_ThicknessSpin">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="15" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(200, 80, 0, 255), stop:1 rgba(170, 50, 0, 255));
color: rgb(255, 255, 255);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <spacer name="m_BottomVerticalSpace">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_MaskValueLabel">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Mask Value:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_DefaultMaterialLabel">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Use Default Material Properties?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="m_DefaultMaterialCheck">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_YoungsModulusLabel">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Young's Modulus:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="13" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_DensityHeatLabel">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Density Heat Product:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="13" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QDoubleSpinBox" name="m_DensityHeatSpin">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QComboBox" name="m_MaskValueCombo">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>(Default)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FEMMaterialsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FEMMaterialsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_PoissonsSpin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FEMMaterialsDialog</receiver>
   <slot>SlotUpdateMaterial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_CrossSectionalSpin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FEMMaterialsDialog</receiver>
   <slot>SlotUpdateMaterial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_ThicknessSpin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FEMMaterialsDialog</receiver>
   <slot>SlotUpdateMaterial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_DensityHeatSpin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FEMMaterialsDialog</receiver>
   <slot>SlotUpdateMaterial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_MaskValueCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FEMMaterialsDialog</receiver>
   <slot>SlotMaskValueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_DefaultMaterialCheck</sender>
   <signal>stateChanged(int)</signal>
   <receiver>FEMMaterialsDialog</receiver>
   <slot>SlotUseDefaultChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_DefaultMaterialCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>FEMMaterialsDialog</receiver>
   <slot>SlotUseDefaultChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_YoungsModulusMantissaSpin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FEMMaterialsDialog</receiver>
   <slot>SlotUpdateMaterial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_YoungsModulusExponentSpin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>FEMMaterialsDialog</receiver>
   <slot>SlotUpdateMaterial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SlotUpdateMaterial()</slot>
  <slot>SlotMaskValueChanged()</slot>
  <slot>SlotUseDefaultChanged(bool)</slot>
 </slots>
</ui>
